docs/content/rn-intro-con.adoc

```diff
@@ -1,6 +1,11 @@
+[id='concept-explanation-{context}']
+= Overview of {PRODUCT}
 
-[id='rn-intro-con']
+{PRODUCT} enables your team to develop, build and deploy enterprise integration solutions on a private Kubernetes cluster managed by Red Hat. 
+{PRODUCT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components. 
+{PRODUCT} enables the operations team focus on high value objectives while Red Hat manage the infrastructure. All the applications and services are protected by Red Hat Single Sign-On.
```

Comment:
I would recommend removing the Operations team focus part of this description (line 6, I don't think it helps with understanding of the service at the highest level and this detail should be described later.

Reply:
@jasonmadigan  I kinda agree with @catrobson  but the email thread on this subject is promoting the Ops capability, WDYT?